# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderCompositor.h
# Commit: 72d4bf5b093e
# Full Hash: 72d4bf5b093e9b00eed78bf4e51850d56df5ae6c
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-05 16:09:03
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add RenderTextureHostSWGL support to D3D11 RenderTextureHosts. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95799
# ==============================================================================

diff -r 99b2e5c55a25 -r 72d4bf5b093e gfx/webrender_bindings/RenderCompositor.h
--- a/gfx/webrender_bindings/RenderCompositor.h	Thu Nov 05 09:14:52 2020 +0000
+++ b/gfx/webrender_bindings/RenderCompositor.h	Thu Nov 05 09:15:05 2020 +0000
@@ -30,6 +30,8 @@
 
 namespace wr {
 
+class RenderCompositorD3D11SWGL;
+
 class RenderCompositor {
  public:
   static UniquePtr<RenderCompositor> Create(
@@ -85,6 +87,10 @@
     return layers::WebRenderCompositor::DRAW;
   }
 
+  virtual RenderCompositorD3D11SWGL* AsRenderCompositorD3D11SWGL() {
+    return nullptr;
+  }
+
   // True if AttachExternalImage supports being used with an external
   // image that maps to a RenderBufferTextureHost
   virtual bool SupportsExternalBufferTextures() const { return false; }